packages/settings-metrics/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "settings-metrics"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
+
+[package.metadata.build-package]
+source-groups = [
+    "settings-extensions/metrics"
+]
+
+# RPM BuildRequires
+[build-dependencies]
+glibc = { path = "../glibc" }
+
+# RPM Requires
+[dependencies]
```

nit: we have a low tolerance for missing newlines at the ends of files... Sorry!

```suggestion
[dependencies]

```